text/0000-system-node.md

```diff
@@ -0,0 +1,108 @@
+
+Maintaining additional Node installations alongside a system Node is especially problematic. For one, it's impossible for the second installation to override the system Node without placing third-party binaries _extremely_ early in the `PATH`, which carries the same security risks mentioned above. For two, even in this scenario, the user experience of global npm package binaries gets muddled: a global binary installed with the system Node may still get picked up if it hasn't been installed as a global in the current Node.
+
+For all the above reasons, **the best approach to using a Node version manager is to avoid having any other Node installations on the system**. Rather than attempting to hide the risks associated with co-installing along with a system Node, we should clearly recommend to users of Notion that if they have an existing system installation of Node, they should uninstall it first.
+
+## Unix and Windows
+
+In Unix OSes, we may want to consider still allowing the ability to opt into overriding a system installation. But since this incurs greater security risk and is generally less reliable, it should not be the recommended path. The installation steps should ask for confirmation and should recommend aborting and choose this as the default action.
+
+In Windows, the system `Path` variable takes precedence over the user `Path` variable---not dissimilar from how Unix environments place `/usr/bin` towards the beginning of `PATH`. But unlike Unix, Windows provides no general mechanism to circumvent this (whereas Unix users can simply prepend new directories to the front of `PATH`). This suggests that the Windows design has a stronger expectation that system installations take precedence over user installations.
+
+Also unlike Unix, Windows developers do not expect environment variables to be configured by terminal bootup scripts. Windows users should be able to expect use cases like executing Node from "Start | Run" or in IDEs to work, which would not work with terminal configuration scripts.
+
+Rather than attempting to circumvent standard practice or provide partial solutions that don't work for all expected contexts, the Windows Notion installer should simply fail if a system Node is detected.
+
+## Detecting system changes
+
+Detecting a system Node at installation time is insufficient since users or system administrators could install another Node installation on the system at any time. So we also need to provide users with the tools to detect these kinds of issues at any time after Notion has already been installed.
+
+# Pedagogy
+[pedagogy]: #pedagogy
+
+The simplest story to explain to users is: **don't install Notion alongside other Node installations**.
```

Comment:
There should be a way for someone to try out Notion even if they are currently using a different version manager, without installing Notion globally (or at the beginning of the path). Possibly some way we can detect that there is another version manager installed, and give some messaging about how to disable the other one, or even automatically disable the other one.